packages/grafana-ui/src/components/Table/styles.ts

```diff
@@ -33,19 +33,16 @@ export const getTableStyles = (theme: GrafanaTheme2) => {
       }
 
       &:hover {
-        overflow: visible;
-        width: auto !important;
         box-shadow: 0 0 2px ${theme.colors.primary.main};
         background: ${background ?? rowHoverBg};
-        z-index: 1;
-
-        .cell-filter-actions  {
-          display: inline-flex;
-        }
```

Comment:
not sure spreading out the styles into multiple files here helps, having them here in one place feels more maintainable, (mainly just the issue of having some styles here like hover style, but some hover styles in another file). There was also the performance optimization of having styles built and fetched once at the root Table component, instead of having to fetch it and cache it for each cell